paywall/src/data-iframe/blockchainHandler/linkKeysToLocks.js

```diff
@@ -13,10 +13,10 @@ export default async function linkKeysToLocks({
   return lockArray.reduce(
     (allLocks, lock) => ({
       ...allLocks,
-      [lock.address]: {
+      [lock.address ? lock.address.toLowerCase() : lock.address]: {
```

Comment:
Would it be a bit more readable to hoist these ternaries to the beginning of the function and declare the address as a variable? Not a blocker, I'll leave it up to you.

Reply:
yeah I think so, it's awfully verbose as-is